core/store/migrate/migrations/0222_jobs_stream_id.sql

```diff
@@ -0,0 +1,6 @@
+-- +goose Up
+ALTER TABLE jobs ADD COLUMN stream_id BIGINT UNIQUE;
```

Comment:
Does `UNIQUE` only apply to non-null values?

Reply:
IIRC it doesn't

@samsondav You might be missing a "WHERE NOT NULL" here.

Reply:
Oh nice! I'm used to pre-pg15 then